app/poi_api/poi_rest_handler.py

```diff
@@ -26,7 +26,7 @@ def get_POI(poi_id):
 
 def suggest_new_POI(poi_suggestion):
     try:
-        return jsonify(poi_service_class.suggest_new_POI(poi_suggestion)), 204
+        return poi_service_class.suggest_new_POI(poi_suggestion)

```

Comment:
If this is returning a 204 with no body you'd want something like.
```suggestion
        poi_service_class.suggest_new_POI(poi_suggestion)
        return None, 204

```